# **ASSIGNMENT**

**Q1: What are the Probability Mass Function (PMF) and Probability Density Function (PDF)? Explain with an example.**

Probability Mass Function (PMF) and Probability Density Function (PDF) are both mathematical functions used to describe the probability distribution of a random variable.

1. Probability Mass Function (PMF):
The PMF is used for discrete random variables. It gives the probability of each possible outcome in a discrete set of values. The PMF is defined as:

PMF(x) = P(X = x)

where X is the random variable and x is a specific value of X. The PMF assigns a probability to each value of the random variable.

Example: Consider rolling a fair six-sided die. The random variable X represents the outcome of the roll. The PMF for X is given by:

PMF(x) = 1/6, for x = 1, 2, 3, 4, 5, 6
PMF(x) = 0,   otherwise

This means that the probability of rolling a 1, 2, 3, 4, 5, or 6 is 1/6 each, and the probability of any other value is 0.

2. Probability Density Function (PDF):
The PDF is used for continuous random variables. It represents the density of probability over a continuous range of values. Unlike the PMF, which assigns probabilities to specific values, the PDF gives the relative likelihood of the random variable taking on a particular value within a range. The PDF is defined such that the probability of the random variable falling within a specific interval is given by the integral of the PDF over that interval.

Example: Let's consider a continuous random variable X that follows a standard normal distribution with mean 0 and standard deviation 1. The PDF for X is given by:

PDF(x) = (1 / √(2π)) * e^((-x^2)/2)

The PDF represents the shape of the bell curve associated with the normal distribution. The height of the curve at any given point represents the relative likelihood of the random variable taking on that value.

It's important to highlight that the sum of probabilities over all possible values for the PMF is equal to 1, whereas the integral of the PDF over its entire range is equal to 1.

**Q2: What is Cumulative Density Function (CDF)? Explain with an example. Why CDF is used?**

The Cumulative Distribution Function (CDF) is a mathematical function that provides the probability that a random variable takes on a value less than or equal to a given value. It gives us cumulative information about the distribution of a random variable.

For a random variable X, the CDF is defined as:

CDF(x) = P(X ≤ x)

The CDF provides the cumulative probabilities for different values of x, starting from negative infinity up to x.

Example: Let's consider a continuous random variable X that follows a standard normal distribution with mean 0 and standard deviation 1. The CDF for X, denoted as Φ(x), can be calculated using the standard normal distribution table or a mathematical function. For instance, Φ(0) represents the probability that X is less than or equal to 0.

The CDF is used to answer questions like "What is the probability that X is less than or equal to 2?" or "What is the probability that X is greater than -1?" by evaluating the CDF at the specific value or range of values of interest. It provides a cumulative perspective on the probabilities associated with a random variable.

The CDF is widely used in statistical analysis and probability theory for various purposes:

1. Probability calculations: The CDF allows us to determine the probability of a random variable falling within a certain range or being less than or equal to a specific value.

2. Quantile calculations: The CDF enables us to find the value(s) of a random variable that correspond to a given probability. This is useful for determining percentiles or confidence intervals.

3. Distribution comparisons: The CDF can be used to compare different distributions and assess their similarities or differences.

4. Random number generation: The CDF can be inverted to generate random numbers from a specific probability distribution.

Overall, the CDF provides valuable information about the cumulative probabilities associated with a random variable, allowing for various statistical calculations and analyses.

**Q3: What are some examples of situations where the normal distribution might be used as a model? Explain how the parameters of the normal distribution relate to the shape of the distribution.**

The normal distribution, also known as the Gaussian distribution or bell curve, is widely used as a model in various fields due to its versatility and applicability to many real-world situations. Here are some examples of situations where the normal distribution might be used as a model:

1. Heights and weights: The distribution of heights and weights in a population often follows a normal distribution. The mean and standard deviation of the normal distribution can provide insights into the average height or weight and the spread of values around the mean.

2. IQ scores: Intelligence quotient (IQ) scores are often assumed to follow a normal distribution. The mean and standard deviation of the normal distribution can help understand the average intelligence level and the variability of IQ scores in a population.

3. Measurement errors: In many scientific and engineering measurements, there is a certain degree of measurement error present. Assuming that these errors are normally distributed allows for the use of statistical techniques that rely on the normal distribution, such as hypothesis testing or confidence intervals.

4. Financial markets: In finance, the normal distribution is commonly used to model the returns of stocks or other financial instruments. This assumption is made in many models, including the famous Black-Scholes option pricing model. The mean and standard deviation of the normal distribution can provide insights into the average return and the volatility of an asset.

The shape of the normal distribution is determined by two parameters: the mean (μ) and the standard deviation (σ). Here's how these parameters relate to the shape of the distribution:

1. Mean (μ): The mean determines the center or location of the normal distribution. It represents the average value around which the data is symmetrically distributed. The mean is also the peak of the bell curve. Shifting the mean to the left or right changes the center of the distribution without affecting its shape.

2. Standard deviation (σ): The standard deviation controls the spread or variability of the distribution. A smaller standard deviation results in a narrower and taller curve, indicating less dispersion of values around the mean. Conversely, a larger standard deviation leads to a wider and flatter curve, indicating greater dispersion.

Therefore, the normal distribution is commonly used in situations where data tends to cluster around a central value with decreasing likelihood as values move away from the center. The mean determines the center of the distribution, and the standard deviation determines the spread or variability of the data.

**Q4: Explain the importance of Normal Distribution. Give a few real-life examples of Normal Distribution.**

The normal distribution is of great importance in statistics and data analysis due to its numerous properties and wide applicability. Here are some key reasons why the normal distribution is important:

Central Limit Theorem: The normal distribution plays a fundamental role in the Central Limit Theorem, which states that the distribution of the sum or average of a large number of independent and identically distributed random variables tends to be approximately normal, regardless of the shape of the original distribution. This theorem is crucial in many statistical inference methods, allowing us to make reliable conclusions about populations based on sample data.

Statistical inference: Many statistical methods and tests, such as hypothesis testing and confidence intervals, rely on the assumption of normality. By assuming that data follows a normal distribution, we can apply these methods to make valid inferences about population parameters.

Parameter estimation: In various statistical models, the assumption of normality simplifies parameter estimation. Maximum likelihood estimation (MLE) and least squares estimation are commonly used techniques that rely on the assumption of normally distributed errors or residuals.

Data transformations: The normal distribution provides a baseline for data transformation. In cases where the data does not follow a normal distribution, applying appropriate transformations (such as logarithmic or power transformations) can help achieve normality and improve the validity of statistical analyses.

Real-life examples of phenomena that often follow a normal distribution include:

Human characteristics: Heights, weights, IQ scores, blood pressure readings, and other physical or psychological traits of human populations often approximate a normal distribution.

Test scores: In standardized tests like the SAT or IQ tests, scores are often assumed to be normally distributed. This assumption helps determine percentiles and set cutoff scores for different performance categories.

Errors in measurements: Measurement errors in various scientific experiments, such as laboratory measurements or instrument readings, are often modeled as normally distributed. This assumption allows for the use of statistical techniques to quantify and analyze measurement uncertainty.

Stock market returns: Daily or monthly returns of stocks or other financial instruments are commonly assumed to follow a normal distribution (or a closely related distribution, such as the log-normal distribution). This assumption is used in various financial models and risk management practices.

These examples highlight the prevalence of the normal distribution in many areas of study and its usefulness in statistical analysis and modeling.





**Q5: What is Bernaulli Distribution? Give an Example. What is the difference between Bernoulli
Distribution and Binomial Distribution?**

The Bernoulli distribution is a discrete probability distribution that models a single binary outcome, which can take one of two possible values, typically labeled as "success" (usually denoted as 1) or "failure" (usually denoted as 0). The distribution is characterized by a single parameter, usually denoted as p, which represents the probability of success.

The probability mass function (PMF) of the Bernoulli distribution is given by:

P(X = k) = p^k * (1-p)^(1-k)

where X is the random variable representing the outcome, k is the value (0 or 1) that X can take, and p is the probability of success.

Example: Let's consider flipping a fair coin. If we define a success as getting heads (H), and failure as getting tails (T), then the outcome of a single coin flip can be modeled using the Bernoulli distribution. The probability of success (getting heads) is 0.5, and the probability of failure (getting tails) is also 0.5. Thus, the PMF of the Bernoulli distribution for this example is:

P(X = 1) = 0.5
P(X = 0) = 0.5

The Bernoulli distribution is a special case of the binomial distribution, which is a discrete probability distribution that models the number of successes in a fixed number of independent Bernoulli trials.

The key difference between the Bernoulli distribution and the binomial distribution lies in the number of trials. In the Bernoulli distribution, there is only one trial, whereas the binomial distribution involves multiple trials.

The binomial distribution is characterized by two parameters: n (the number of trials) and p (the probability of success in each trial). The binomial distribution describes the probability of obtaining exactly k successes in n independent Bernoulli trials.

The probability mass function (PMF) of the binomial distribution is given by:

P(X = k) = C(n, k) * p^k * (1-p)^(n-k)

where X is the random variable representing the number of successes, k is the number of successes, n is the number of trials, p is the probability of success in each trial, and C(n, k) represents the number of ways to choose k successes out of n trials (binomial coefficient).

Therefore, the Bernoulli distribution models a single binary outcome, while the binomial distribution models the number of successes in a fixed number of independent Bernoulli trials. The Bernoulli distribution can be seen as a special case of the binomial distribution with n = 1.

**Q6. Consider a dataset with a mean of 50 and a standard deviation of 10. If we assume that the dataset
is normally distributed, what is the probability that a randomly selected observation will be greater
than 60? Use the appropriate formula and show your calculations.**

Calculation by python:

In [1]:
import statistics,scipy

In [2]:
scipy.stats.norm.cdf(60,50,10)

0.8413447460685429

In [3]:
prob=1-scipy.stats.norm.cdf(60,50,10)

In [4]:
print("Probability that a randomly selected observation will be greater than 60:",prob*100)

Probability that a randomly selected observation will be greater than 60: 15.865525393145708


Calculation by hand:

Given: μ=50<br>
       σ=10<br>
       x=60<br>
       P(X > x) = 1 - P(X ≤ x) = 1 - CDF(x; μ, σ)<br>
       
Firstly, we will find CDF(x; μ, σ):<br>       
       
If we are given the mean (μ),standard deviation (σ), and a specific value (x) then to calculate the CDF in this scenario:

1. Standardize the value x: Subtract the mean (μ) from x and divide the result by the standard deviation (σ).

   z = (x - μ) / σ

3. Use the standardized value z to look up the corresponding cumulative probability in a standard normal distribution table or use a statistical software function that provides the CDF of a standard normal distribution (e.g., the erf or norm.cdf function in Python).

   F(z) = P(Z ≤ z)

   Here, Z represents a standard normal random variable.

4. The calculated value F(z) represents the probability that a standard normal random variable is less than or equal to z, which is equivalent to the probability that the original random variable X is less than or equal to x.       
       
       

z = (x - μ) / σ
          =(60-50)/10
          =1<br>
          
Then, F(z) = P(Z ≤ z) 
         = 0.8413<br>
           
Thus, P(X <= x) =F(z) <br> 
Therefore, P(X <= x)=0.8413<br>

Hence, P(X > x) = 1 - P(X ≤ x) = 1 - CDF(x; μ, σ)
                = 1-0.8413
                = 0.15865<br>
                
Probability that a randomly selected observation will be greater than 60 is **15.865**             

**Q7: Explain uniform Distribution with an example.**

The uniform distribution is a probability distribution that describes a situation where all outcomes within a given range are equally likely. It is characterized by a constant probability density function (PDF) over the range of possible values.

In the uniform distribution, every possible outcome has the same probability of occurring. This distribution is often visualized as a rectangular shape, where the height of the rectangle represents the constant probability.

Example: Let's consider rolling a fair six-sided die. The outcomes of rolling the die are integers from 1 to 6. If each face of the die is equally likely, we can model the outcome as a uniform distribution.

In this case, the probability of getting any specific face (1, 2, 3, 4, 5, or 6) is 1/6, as each face is equally likely. The uniform distribution for this example can be represented by the following probability density function (PDF):

f(x) = 1/6, for x ∈ {1, 2, 3, 4, 5, 6}
f(x) = 0, otherwise

This means that any value outside the range of 1 to 6 will have a probability of 0, while the probability of getting any specific value within that range is 1/6.

The uniform distribution is not limited to discrete outcomes like rolling a die. It can also apply to continuous outcomes. For example, if you randomly select a number between 0 and 1 with equal probability, that would follow a continuous uniform distribution over the interval [0, 1].

The uniform distribution is used in various applications, such as random number generation, simulations, and when assuming equal likelihood for certain events or measurements.

**Q8: What is the z score? State the importance of the z score.**

The z-score, also known as the standard score, is a statistical measure that quantifies how many standard deviations a data point is away from the mean of a distribution. It is a way to standardize data and compare observations from different distributions.

The formula to calculate the z-score for a data point x, given the mean (μ) and standard deviation (σ) of the distribution, is:

z = (x - μ) / σ

The z-score tells us how many standard deviations a data point is above or below the mean. A positive z-score indicates that the data point is above the mean, while a negative z-score indicates it is below the mean. A z-score of 0 means the data point is exactly at the mean.

Importance of the z-score:

1. Standardization and Comparison: The z-score allows for the standardization of different variables or data sets, making it easier to compare observations from different distributions. It provides a common scale to assess the relative position of a data point within its distribution.

2. Identifying Outliers: The z-score helps identify outliers by quantifying how extreme or unusual a data point is relative to the rest of the distribution. Data points with z-scores significantly above or below a certain threshold (e.g., ±2 or ±3) are considered outliers.

3. Probability Calculation: The z-score is used to calculate probabilities associated with a normal distribution. By converting a value to its corresponding z-score, we can determine the probability of obtaining a value less than, greater than, or between certain values.

4. Hypothesis Testing: The z-score is essential in hypothesis testing and constructing confidence intervals. It allows researchers to determine whether an observed difference or effect is statistically significant by comparing the z-score to critical values from the standard normal distribution.

5. Data Transformation: The z-score transformation is often employed to normalize skewed or non-normally distributed data. By converting data to z-scores, skewed distributions can be made closer to a standard normal distribution, which can be useful for certain statistical analyses.

Overall, the z-score is a fundamental statistical tool that provides a standardized measure of a data point's deviation from the mean. It enables comparison, identification of outliers, probability calculations, hypothesis testing, and data transformation.

**Q9: What is Central Limit Theorem? State the significance of the Central Limit Theorem.**

The Central Limit Theorem (CLT) is a fundamental concept in statistics that states that the sampling distribution of the mean of a sufficiently large number of independent and identically distributed (i.i.d.) random variables will be approximately normally distributed, regardless of the shape of the original population distribution.

In simpler terms, the Central Limit Theorem states that if you take many random samples from any population, the distribution of the sample means will be approximately normal, regardless of the shape of the population distribution.

Significance of the Central Limit Theorem:

1. Normal Approximation: The Central Limit Theorem allows us to approximate the distribution of sample means, regardless of the shape of the population distribution. This approximation is particularly useful when the population distribution is not known or is non-normal.

2. Inference and Confidence Intervals: The CLT is the foundation of statistical inference and hypothesis testing. It enables the use of parametric tests, such as the t-test or z-test, which rely on the assumption of normality. It also helps in constructing confidence intervals for population parameters, such as the mean.

3. Sample Size Determination: The CLT helps determine the appropriate sample size required for statistical inference. By knowing the desired level of precision and the variability of the population, we can estimate the sample size needed to achieve a desired level of confidence in our results.

4. Real-World Applications: The Central Limit Theorem is applicable in various fields and practical scenarios. It allows us to make inferences about large populations using samples, even when the population distribution is unknown or non-normal. This is particularly useful in market research, quality control, opinion polling, and many other areas where sampling and generalization are crucial.

5. Basis for Normality Assumption: Many statistical techniques, such as linear regression and analysis of variance (ANOVA), assume that the data are normally distributed. The Central Limit Theorem provides a justification for these assumptions, as the means of samples tend to follow a normal distribution.

Therefore, the Central Limit Theorem is significant because it enables us to make reliable statistical inferences, perform hypothesis testing, estimate confidence intervals, and apply various statistical techniques even when the population distribution is unknown or non-normal.

**Q10: State the assumptions of the Central Limit Theorem.**

The Central Limit Theorem (CLT) relies on certain assumptions to hold true. These assumptions are as follows:

1. Independence: The random variables in the sample should be independent of each other. This means that the value of one random variable should not be influenced by or dependent on the values of other random variables in the sample.

2. Identical Distribution: The random variables in the sample should be identically distributed. This means that they should follow the same probability distribution, with the same mean and variance.

3. Finite Variance: The random variables in the population should have a finite variance. This ensures that the sample means are well-behaved and do not have extreme or infinite values.

4. Sufficient Sample Size: The Central Limit Theorem becomes more applicable and accurate as the sample size increases. Although there is no specific threshold, a commonly used guideline is that the sample size should be at least 30. However, the CLT can still provide reasonable approximations for moderately sized samples, especially when the underlying population distribution is not heavily skewed or has outliers.

It is important to note that violating these assumptions may affect the applicability and accuracy of the Central Limit Theorem. If the assumptions are not met, alternative statistical techniques or modifications to the CLT may be required to make valid inferences.


-------------